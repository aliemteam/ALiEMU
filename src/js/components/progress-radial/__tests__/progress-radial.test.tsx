import React from 'react';
import renderer from 'react-test-renderer';

import ProgressRadial from '../index';

it('renders correctly', () => {
    const tree = renderer.create(
        <ProgressRadial diameter={50} thickness={5} max={100} value={50} />,
    );
    expect(tree).toMatchSnapshot();
});
