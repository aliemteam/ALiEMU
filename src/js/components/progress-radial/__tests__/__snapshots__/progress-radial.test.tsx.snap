// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  aria-valuemax={100}
  aria-valuemin={0}
  aria-valuenow={50}
  className="container"
  role="progressbar"
>
  <svg
    className="no-inherit circle"
    height={50}
    viewBox="0 0 50 50"
    width={50}
  >
    <circle
      className="circleBackground"
      cx={25}
      cy={25}
      r={22.5}
      strokeWidth={5}
    />
    <circle
      className="circleFill"
      cx={25}
      cy={25}
      r={22.5}
      strokeDasharray={141.3716694115407}
      strokeWidth={5}
      style={
        Object {
          "strokeDashoffset": 70.68583470577035,
        }
      }
    />
  </svg>
</div>
`;
